contracts/WildcardsQV.sol

```diff
@@ -40,6 +40,9 @@ contract WildcardsQV is Initializable {
     uint256 public currentWinner;
     uint256 public totalVotes;
 
+    // if a payment fails, the user will be able to pull the amount stored here:
+    mapping(address => uint256) public failedTransferCredits;
```

Comment:
Is there a function for them to withdraw these credits?

Reply:
Yes, I'll comment on it.